src/Moryx.Resources.Management/Resources/ResourceEntityAccessor.cs

```diff
@@ -78,8 +78,7 @@ public Resource Instantiate(IResourceTypeController typeController, IResourceGra
 
             // Read everything else from defaults
             ValueProviderExecutor.Execute(Instance, new ValueProviderExecutorSettings()
-                .AddFilter(new DataMemberAttributeValueProviderFilter(false))
-                .AddDefaultValueProvider());
+                .AddFilter(new DataMemberAttributeValueProviderFilter(false)));
```

Comment:
I am wondering whether we need this call at all now. Does it still do anything without a registered provider? Does it populate `null` objects for example?

Reply:
Really is not needed anymore, since it should do nothing